packages/terra-toggle-button/tests/wdio/toggle-button-spec.js

```diff
@@ -0,0 +1,195 @@
+/* global before, browser, Terra */
+
+describe('Toggle Button', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-toggle-button/toggle-button/default-toggle-button'));
+
+    // For consistency with test below
+    it('should show a closed button', () => {
+      const screenshot = browser.checkElement('.button');
+      expect(screenshot).to.matchReference('withinTolerance');
+    });
+
```

Comment:
add `Terra.should.beAccessible();` here so we are ensure the open and closed state are both accessible